src/components/Bootstrap/Navbar/ZekBvNavbar.vue

```diff
@@ -18,16 +18,14 @@
       <BNavbarToggle target="nav-collapse" />
       <BCollapse id="nav-collapse" is-nav>
         <BNavbarNav v-for="(item, index) in items" :key="index" :class="item.navClass">
-          <component :is="tabType[item.type]" v-bind="item"
-            >{{ item.type != 'dropdown' ? item.text : '' }}
-            <template v-if="item.type == 'dropdown'"
-              ><BDropdownItem
-                v-for="(option, index) in item.options"
-                :key="index"
-                :href="option.link"
-                >{{ option.text }}</BDropdownItem
-              ></template
-            >
+          <ZekBvDropdown
+              v-if="item.type == 'dropdown'"
+              :items="item.options"
+              v-bind="item.dropdownProps"
+              v-on="item.dropdownEvents"
+            ></ZekBvDropdown>
+          <component v-else :is="tabType[item.type]" v-bind="item"
+            >{{ item.text }}
```

Comment:
Why not merge these 2 together and reference the ZekBvDropdown in the `tabType` map?